templates/ords-remix-jwt-sample/ords/openAPI Catalog/concert_app.euser.v1.json

```diff
@@ -1,1124 +1,1124 @@
 {
     "openapi": "3.0.0",
     "info": {
-      "title": "ORDS generated API for concert_app.euser.v1",
-      "version": "1.0.0",
-      "description": "end user APIs Version 1"
+        "title": "ORDS generated API for concert_app.euser.v1",
+        "version": "1.0.0",
+        "description": "end user APIs Version 1"
     },
     "servers": [
-      {
-        "url": "https://g0ceab9035f2a02-adboiram.adb.ca-toronto-1.oraclecloudapps.com/ords/concert_sample_app/euser/v1/"
-      }
+        {
+            "url": "https://g389fa1f14b6c7b-ckhclwyq6qtbgq3a.adb.ca-toronto-1.oraclecloudapps.com/ords/frank/euser/v1/"
```

Comment:
need to manually test this, the diff is too big, also, now that this endpoint is public we should remove any reference to our test instances, just in case.

Reply:
Diff updated.